test/parallel/rolldown.withdraw.merkleRoot.test.ts

```diff
@@ -0,0 +1,472 @@
+  let error: any;
+  //since there is no token in the Arbitrum chain by default, we create a new one
+  const minToBeSequencer = await SequencerStaking.minimalStakeAmount();
+  const blocksForSequencerUpdate =
+    await SequencerStaking.blocksForSequencerUpdate();
+  await clearActiveSequencers();
+  await signTx(
+    await getApi(),
+    await SequencerStaking.provideSequencerStaking(
+      minToBeSequencer.addn(1234),
+      "Arbitrum",
+    ),
+    testUser.keyRingPair,
+  );
+  await Rolldown.waitForReadRights(
+    testUser.keyRingPair.address,
+    50,
+    "Arbitrum",
+  );
+  const txIndex = await Rolldown.lastProcessedRequestOnL2("Arbitrum");
+  const api = getApi();
+  const update = new L2Update(api)
+    .withDeposit(
+      txIndex,
+      testUser.keyRingPair.address,
+      testUser.keyRingPair.address,
+      BN_MILLION,
+    )
+    .on("Arbitrum")
+    .buildUnsafe();
+  await signTx(api, update, testUser.keyRingPair).then((events) => {
+    const res = getEventResultFromMangataTx(events);
+    expect(res.state).toEqual(ExtrinsicResult.ExtrinsicSuccess);
+  });
+  await waitForNBlocks(blocksForSequencerUpdate);
+
+  //we need to run <batchSize> extrinsic to update the start of the automatic batching period
+  await Sudo.batchAsSudoFinalized(
+    ...(await getWithdrawalExtrinsics(batchSize)),
+  );
+
+  await Sudo.batchAsSudoFinalized(
+    ...(await getWithdrawalExtrinsics(batchSize - 1)),
+  );
+
+  await signTx(
+    getApi(),
+    await Withdraw(
+      testUser,
+      BN_HUNDRED,
+      testUser.keyRingPair.address,
+      "Arbitrum",
+    ),
+    testUser.keyRingPair,
+  ).then((events) => {
+    const res = getEventResultFromMangataTx(events);
+    expect(res.state).toEqual(ExtrinsicResult.ExtrinsicSuccess);
+  });
+  try {
+    await Rolldown.waitForNextBatchCreated("Ethereum", 10);
+  } catch (err) {
+    error = err;
+  }
+  expect(error).toEqual(
+    "method rolldown.TxBatchCreated not found within blocks limit",
+  );
+});
+
+describe("Pre-operation withdrawal tests", () => {
+  beforeEach(async () => {
+    await Sudo.batchAsSudoFinalized(
+      ...(await getWithdrawalExtrinsics(batchSize)),
+    );
+    const event = await Rolldown.waitForNextBatchCreated(
+      "Ethereum",
+      waitingBatchPeriod,
+    );
+    expect(event.source).toEqual("AutomaticSizeReached");
+  });
+
+  test("Given <batchSize> withdrawals WHEN they run successfully THEN a batch is generated AUTOMATICALLY from that L1, from ranges of (n,n+<batchSize>-1)", async () => {
+    const nextRequestId = JSON.parse(
+      JSON.stringify(await api.query.rolldown.l2OriginRequestId()),
+    );
+    await Sudo.batchAsSudoFinalized(
+      ...(await getWithdrawalExtrinsics(batchSize)),
+    );
+    const event = await Rolldown.waitForNextBatchCreated(
+      "Ethereum",
+      waitingBatchPeriod,
+    );
+    const sequencersList = await SequencerStaking.activeSequencers();
+    expect(sequencersList.toHuman().Ethereum).toContain(event.assignee);
```

Comment:
why this?

Reply:
Because each user can activate withdrawal and the user would be the assignee for this operation